CHANGELOG

```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* Fix concurrency bug which can lead to lost threads. See BTS-2087.
```

Comment:
Maybe we can improve this slightly to make it more useful to users, like

> BTS-2087: Fix a concurrency bug that can lead the cleanup of an AQL query to hang and block a thread.

or so?